src/thunderbots/software/ai/hl/stp/evaluation/enemy_threat.h

```diff
@@ -43,4 +89,36 @@ namespace Evaluation
     std::optional<std::pair<int, std::optional<Robot>>> getNumPassesToRobot(
         const Robot &initial_passer, const Robot &final_receiver,
         const Team &passing_team, const Team &other_team);
+
+    /**
+     * Sorts the given list of threats in order of decreasing threat, so the
+     * "most threatening threat" will be at the front of the vector, and the
+     * "least threatening threat" will be at the end of the vector
+     *
+     * @param threats The given list of threats to sort
+     * @return A new list of threats where the threats are sorted in decreasing order
+     */
+    std::vector<Evaluation::EnemyThreat> sortThreatsInDecreasingOrder(
+        std::vector<Evaluation::EnemyThreat> threats);
```

Comment:
Minor thought.
Would make sense to rename 'threats' to something like 'unsorted_threats'?

Reply:
Technically there's no guarantee the given list is unsorted (it could already be sorted), so I think it's fine to leave.